react-native/android/build.gradle

```diff
@@ -31,10 +31,17 @@ import org.apache.tools.ant.filters.ReplaceTokens
 // We download various C++ open-source dependencies into downloads.
 // We then copy both the downloaded code and our custom makefiles and headers into third-party-ndk.
 // After that we build native code from src/main/jni with module path pointing at third-party-ndk.
+def npmExecutable = 'npm';
 
-ext.coreVersion = "npm --silent run get-core-version".execute().text.trim()
-ext.syncVersion = "npm --silent run get-sync-version".execute().text.trim()
-def currentVersion = "npm --silent run get-version".execute().text.trim()
+if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        npmExecutable = 'npm.cmd'
+}
+ext.coreVersion = "$npmExecutable --silent run get-core-version".execute().text.trim()
```

Comment:
Also If we can remove the symlinks in the repo it will remove the need to do this pre/post build actjions which will be way more better. Not sure if we can do it easely. 
Currently these are the symlinks

examples/ReactNativeBenchmarks/.eslintrc.json <<===>> ../ReactExample/.eslintrc.json
react-native/android/src/main/jni/src <<===>> ../../../../../src/
react-native/android/src/main/jni/vendor <<===>> ../../../../../vendor
tests/react-test-app/android/app/src/main/assets <<===>> ../../../../../data